package.json

```
@@ -28,6 +28,10 @@
         "ag-grid-community": ">=29.0.0",
         "ag-grid-enterprise": ">=29.0.0",
         "ag-grid-react": ">=29.0.0",
+        "d3-format": ">=3.1.0",
+        "d3-array": ">=3.2.2",
+        "d3-time": ">=3.1.0",
+        "d3-time-format": ">=4.1.0",
```

```suggestion
        "d3-format": "^3.1.0",
        "d3-array": "^3.2.2",
        "d3-time": "^3.1.0",
        "d3-time-format": "^4.1.0",
```
The difference is `^` won't go up a major version so won't accidentally bring in breaking changes, whereas `>=` can bump major versions.